examples/scripts/dpo_online.py

```diff
@@ -81,12 +85,21 @@
         model_config.model_name_or_path, trust_remote_code=model_config.trust_remote_code, **model_kwargs
     )
 
-    reward_model = AutoModelForSequenceClassification.from_pretrained(
-        training_args.reward_model_path,
-        num_labels=1,
-        trust_remote_code=model_config.trust_remote_code,
-        **model_kwargs,
-    )
+    if training_args.reward_model_path is not None:
```

Comment:
WDYT about having a helper method like `get_reward_model(TrainingArguments, ModelConfig)` so we can (a) unit test it and (b) not have duplicate logic across all scripts?

Reply:
#2271